applications/sasquatch/charts/prompt-processing/templates/kafka-topics.yaml

```diff
@@ -1,21 +1,13 @@
-
+{{- $cluster := .Values.cluster.name }}
+{{- range $topic := .Values.topics }}
 ---
-apiVersion: kafka.strimzi.io/v1beta2
+apiVersion: kafka.strimzi.io/v1beta1
```

Comment:
I think the correct API version for Strimzi is v1beta2, any reason for changing it?

Reply:
Thanks for catching.  It was copied from the [control-system](https://github.com/lsst-sqre/phalanx/blob/5966a529189424cca2fc9e42342677254ab413a5/applications/sasquatch/charts/control-system/templates/kafka-topics.yaml#L4) and didn't notice it.  Updated it to `v1beta2`